app/build.gradle

```diff
@@ -82,7 +82,7 @@ android {
     compose true
   }
   composeOptions {
-    kotlinCompilerExtensionVersion compose_version
+    kotlinCompilerExtensionVersion compose_compiler_version
```

Comment:
the compiler is now separated from the compose version which makes kotlin upgrades easier